xml/admin_gui_dashboard.xml

```diff
@@ -2327,6 +2410,130 @@ Oct 04 07:48:53 doc-ses6min4 systemd[1]: Started Ceph rbd mirror daemon.
 </screen>
     </tip>
    </sect3>
+   <sect3 xml:id="cert-sign-custom-CA">
+     <title>Certificates Signed with a Custom CA</title>
+     <para>
+       The following procedure needs to be followed once to create the Root CA.
+     </para>
+     <procedure>
+       <step>
+         <note>
+           <para>
+             This is the key used to sign the certificate requests, anyone
+             holding this can sign certificates on your behalf.
+           </para>
+         </note>
+         <para>
+           Create the Root Key:
+         </para>
+<screen>
+openssl genrsa -des3 -out rootCA.key 4096
+</screen>
+        <note>
+          <para>
+            If you want a non-password protected key, remove the
+            <option>-des3</option> option.
+          </para>
+        </note>
+       </step>
+       <step>
+         <para>
+           Create and self-sign the Root Certificate:
+         </para>
+<screen>
+openssl req -x509 -new -nodes -key rootCA.key -sha256 -days 1024 -out rootCA.crt
+</screen>
+       </step>
+     </procedure>
+     <para>
+       The following procedure needs to be followed for each server
+       that needs a trusted certificate from our CA.
+     </para>
+     <procedure>
+       <step>
+         <para>
+          Create the certificate key:
+        </para>
+<screen>
+openssl genrsa -out mydomain.com.key 2048
+</screen>
+       </step>
+       <step>
+         <para>
+           The certificate signing request is where you specify the details for
+           the certificate you want to generate. This request is processed by
+           the owner of the Root Key to generate the certificate.
+         </para>
+         <important>
+           <para>
+             When creating the signing request, it is important to specify
```

Comment:
You are talking about the **certificate** signing request, right? Maybe be explict?